backend/clickhouse/errors.go

```diff
@@ -552,57 +552,66 @@ func (client *Client) QueryErrorHistogram(ctx context.Context, projectId int, qu
-		modelInputs.ReservedErrorObjectKeyServiceVersion:  "Service.Version",
-		modelInputs.ReservedErrorObjectKeyEnvironment:     "Environment",
+		modelInputs.ReservedErrorObjectKeyTraceID:         "TraceID",
+		modelInputs.ReservedErrorObjectKeyVisitedURL:      "VisitedURL",
 	},
-	BodyColumn:   "Event",
 	ReservedKeys: modelInputs.AllReservedErrorObjectKey,
 }
 
-var errorsJoinedTableConfig = model.TableConfig[modelInputs.ReservedErrorObjectKey]{
+var errorsJoinedTableConfig = model.TableConfig[modelInputs.ReservedErrorsJoinedKey]{
 	TableName: "errors_joined_vw",
-	KeysToColumns: map[modelInputs.ReservedErrorObjectKey]string{
-		modelInputs.ReservedErrorObjectKeyBrowser:        "Browser",
-		modelInputs.ReservedErrorObjectKeyEnvironment:    "Environment",
-		modelInputs.ReservedErrorObjectKeyEvent:          "Event",
-		modelInputs.ReservedErrorObjectKeyHasSessions:    "HasSession",
-		modelInputs.ReservedErrorObjectKeyOs:             "OSName",
-		modelInputs.ReservedErrorObjectKeyServiceName:    "ServiceName",
-		modelInputs.ReservedErrorObjectKeyServiceVersion: "ServiceVersion",
-		modelInputs.ReservedErrorObjectKeyTag:            "ErrorTagTitle",
-		modelInputs.ReservedErrorObjectKeyType:           "Type",
-		modelInputs.ReservedErrorObjectKeyURL:            "VisitedURL",
-		modelInputs.ReservedErrorObjectKeyTimestamp:      "Timestamp",
-		modelInputs.ReservedErrorObjectKeyStatus:         "Status",
+	KeysToColumns: map[modelInputs.ReservedErrorsJoinedKey]string{
+		modelInputs.ReservedErrorsJoinedKeyBrowser:        "Browser",
+		modelInputs.ReservedErrorsJoinedKeyClientID:       "ClientID",
+		modelInputs.ReservedErrorsJoinedKeyEnvironment:    "Environment",
+		modelInputs.ReservedErrorsJoinedKeyEvent:          "Event",
+		modelInputs.ReservedErrorsJoinedKeyHasSession:     "HasSession",
+		modelInputs.ReservedErrorsJoinedKeyOsName:         "OSName",
+		modelInputs.ReservedErrorsJoinedKeyServiceName:    "ServiceName",
+		modelInputs.ReservedErrorsJoinedKeyServiceVersion: "ServiceVersion",
+		modelInputs.ReservedErrorsJoinedKeyTag:            "ErrorTagTitle",
+		modelInputs.ReservedErrorsJoinedKeyType:           "Type",
+		modelInputs.ReservedErrorsJoinedKeyVisitedURL:     "VisitedURL",
+		modelInputs.ReservedErrorsJoinedKeyTimestamp:      "Timestamp",
+		modelInputs.ReservedErrorsJoinedKeyStatus:         "Status",
 	},
 	BodyColumn:   "Event",
-	ReservedKeys: modelInputs.AllReservedErrorObjectKey,
+	ReservedKeys: modelInputs.AllReservedErrorsJoinedKey,
 }
 
-var errorsSampleableTableConfig = sampleableTableConfig[modelInputs.ReservedErrorObjectKey]{
+var errorsSampleableTableConfig = sampleableTableConfig[modelInputs.ReservedErrorsJoinedKey]{
 	tableConfig: errorsJoinedTableConfig,
 	useSampling: func(time.Duration) bool {
 		return false
 	},
 }
 
 func ErrorMatchesQuery(errorObject *model2.BackendErrorObjectInput, filters listener.Filters) bool {
-	return matchesQuery(errorObject, ErrorObjectsTableConfig, filters)
```

Comment:
I want to double check that this would not affect the behavior of filters? Seems like their were additional columns in this config that were removed, but tried to include the relevant ones in  `errorsJoinedTableConfig`. However, maybe we need to map these to the SQL table columns in a new list?